scripts/core/editor2/editor.js

```diff
@@ -1006,6 +1006,14 @@ angular.module('superdesk.apps.editor2', [
                         scope.medium.destroy();
                     }
 
+                    var deregisterTansa = $rootScope.$on('tansa:after', () => {
```

Comment:
We've spent 2 days together on fixing this bug, so it would be nice to document this function and explain why we need to read node `innerHTML` and update model, because this is very unusual. For example:
```js
// Because the tansa spellchecker modifies DOM directly on auto-correct,
// the editor has issues picking up new scope and model values, so we
// have to update these manually.
```
/cc @vied12 maybe has input, if not ignore this